src/oca_github_bot/tasks/tag_ready_to_merge.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) ACSONE SA/NV 2018
+# Copyright (c) https://github.com/madman-bob 2018
+# Distributed under the MIT License (http://opensource.org/licenses/MIT).
+
+from datetime import datetime, timedelta
+
+from .. import github
+from ..github import gh_call
+from ..queue import task, getLogger
+
+_logger = getLogger(__name__)
+
+
+APPROVALS_REQUIRED = 2
```

Comment:
Maybe this value of `2` can be a fallback for a customized value on each repository.
This would allow more flexiblity and this kind" of rule could be managed by the PSC of the repo they are responsible of.

Reply:
The question would be where to configure these parameters per repo. In any case, these label are only advisory.